docs/src/components/Feature.tsx

```diff
@@ -58,7 +58,7 @@ function getGitHubUrlForExample(platform) {
 export function Feature({ name = required('Missing feature name') }) {
   const [source, setSource] = React.useState(null);
   const { pathname, query } = useRouter();
-  const { platform = 'react' } = query;
+  const { platform = 'next' } = query;
```

Comment:
`react` is still our most common framework and should be the default for when `?platform=...` isn't explicitly set:

> https://ui.docs.amplify.aws/ui/components/authenticator#sign-up